doc/tutorial/Server.lhs

```diff
@@ -8,7 +8,15 @@ type. Can we have a webservice already?
 Equipped with some basic knowledge about the way we represent APIs, let's now
 write our first webservice.
 
-The source for this tutorial section is a literate haskell file, so first we
+## Setting up our project
+
+Don't forget to look at the [cabal file included with this example project](doc/tutorial/tutorial.cabal). You can run the main function provided in the following examples within the `Server` module without using a separate executable.
+
+If you're using Stack, then in your `stack.yaml` you may want to specify version 0.11.1.1 for `aeson`, and version 0.5 for `servant` and `servant-server` in your `extra-deps`.
```

Comment:
I'm not sure what you're trying to explain here.

Is this about setting up a new `stack` project for `servant`? (In that case I think this explanation should go somewhere else.)

Or is this about how to set up a stack file so that you can load the tutorial in `ghci`? In that case, you can use the stack file provided in the repo (top-level). So in `doc/tutorial` you can do:

``` bash
chmod go-w .ghci .
stack setup
stack build
stack exec ghci Server.lhs
```

^^^ This information is sorely missing from the tutorial, btw, I think.